ci/install_abseil.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    #cmake not installed, exiting
```

Comment:
Log an error message to the console?

Reply:
good point. Done that

Reply:
I'd say - don't even check for CMake in this script. We have so many other scripts, and we can just say that CMake is a major pre-requisite to get anything built, esp. if you are setting up a build loop for CMake - it is already making assumption that CMake has to be there.